demos/projects/ESPRESSIF/esp32/README.md

```diff
@@ -1,284 +1,304 @@
-I (509) heap_init: At 4009437C len 0000BC84 (47 KiB): IRAM
-I (516) spi_flash: detected chip: winbond
-I (520) spi_flash: flash io: dio
-W (524) spi_flash: Detected size(8192k) larger than the size in the binary image header(2048k). Using the size in the binary image header.
-I (538) cpu_start: Starting scheduler on PRO CPU.
-I (0) cpu_start: Starting scheduler on APP CPU.
-I (732) wifi:wifi driver task: 3ffba728, prio:23, stack:6656, core=0
-I (732) system_api: Base MAC address is not set
-I (732) system_api: read default base MAC address from EFUSE
-I (752) wifi:wifi firmware version: ff5f4ea
-I (752) wifi:wifi certification version: v7.0
-I (752) wifi:config NVS flash: enabled
-I (752) wifi:config nano formating: disabled
-I (762) wifi:Init data frame dynamic rx buffer num: 32
-I (762) wifi:Init management frame dynamic rx buffer num: 32
-I (772) wifi:Init management short buffer num: 32
-I (772) wifi:Init dynamic tx buffer num: 32
-I (782) wifi:Init static rx buffer size: 1600
-I (782) wifi:Init static rx buffer num: 10
-I (782) wifi:Init dynamic rx buffer num: 32
-I (792) wifi_init: rx ba win: 6
-I (792) wifi_init: tcpip mbox: 32
-I (792) wifi_init: udp mbox: 6
-I (802) wifi_init: tcp mbox: 6
-I (802) wifi_init: tcp tx win: 5744
-I (812) wifi_init: tcp rx win: 5744
-I (812) wifi_init: tcp mss: 1440
-I (812) wifi_init: WiFi IRAM OP enabled
-I (822) wifi_init: WiFi RX IRAM OP enabled
-I (822) example_connect: Connecting to ContosoWiFi...
-I (832) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07
-I (932) wifi:mode : sta (84:cc:a8:4c:7e:fc)
-I (932) wifi:enable tsf
-I (942) example_connect: Waiting for IP(s)
-I (2992) wifi:new:<11,0>, old:<1,0>, ap:<255,255>, sta:<11,0>, prof:1
-I (2992) wifi:state: init -> auth (b0)
-I (3002) wifi:state: auth -> assoc (0)
-I (3012) wifi:state: assoc -> run (10)
-I (3022) wifi:connected with ContosoWiFi, aid = 1, channel 11, BW20, bssid = 74:ac:b9:c1:39:76
-I (3022) wifi:security: WPA2-PSK, phy: bgn, rssi: -61
-I (3022) wifi:pm start, type: 1
-
-I (3042) wifi:AP's beacon interval = 102400 us, DTIM period = 3
-W (3042) wifi:<ba-add>idx:0 (ifx:0, 74:ac:b9:c1:39:76), tid:0, ssn:0, winSize:64
-I (3612) esp_netif_handlers: example_connect: sta ip: 192.168.1.220, mask: 255.255.255.0, gw: 192.168.1.1
-I (3612) example_connect: Got IPv4 event: Interface "example_connect: sta" address: 192.168.1.220
-I (3622) example_connect: Connected to example_connect: sta
-I (3622) example_connect: - IPv4 address: 192.168.1.220
-[INFO] [AzureIoTDemo] [sample_azure_iot.c:580] Creating a TLS connection to contoso-iot-hub.azure-devices.net:8883.
-
-I (6322) tls_freertos: (Network connection 0x3ffc8c4c) Connection to contoso-iot-hub.azure-devices.net established.
-[INFO] [AzureIoTDemo] [sample_azure_iot.c:384] Creating an MQTT connection to contoso-iot-hub.azure-devices.net.
-...
-```
\ No newline at end of file
+# Connect an ESPRESSIF ESP32 using Azure IoT Middleware for FreeRTOS

```

Comment:
same here as below with the sample file. Is this changed at all? Why is the diff wonky?